Image3dAPI/IImage3d.idl

```diff
@@ -294,7 +294,7 @@ interface IImage3dSource : IUnknown {
     [helpstring("Retrieve color-map table for mapping image intensities to RGBx values. Length is 256.")]
     HRESULT GetColorMap ([out,retval] SAFEARRAY(unsigned int) * map);
 
-    [helpstring("Get ECG data if available [optional]")]
+    [helpstring("Get ECG data if available [optional]. Shall return S_OK with an empty EcgSeries if EGC is not available.")]
```

Comment:
EcgSeries could meaningfully be used to transfer only trig points, without ECG trace. DICOM files typically only contain the R-wave times, not the trace values. EcgSeries samples could also benefit from a document update to highlight this. 

In addition, it may be cases where a loader returns trig points where some intervals contain no frames. This could be the case for multi-cycle acquisitions, where the full volume is composed of data acquired over multiple cycles. It would then be expected behavior for a loader to return trig intervals without frames. A client that displays QRS/trig points and/or trace could then benefit from trimming the displayed data. The documentation could be updated to highlight this.